src/less/edit.less

```diff
@@ -0,0 +1,67 @@
+.edit-form {
+    /* width: 350px;
```

Comment:
Can we delete these commented out lines?

Reply:
Actually, is this file needed now that you've moved to AntD?

Reply:
yeah. i will use those to do some minor styling